moj-be/modules/custom/moj_resources/src/ContentApiClass.php

```diff
@@ -5,92 +5,194 @@
 use Drupal\node\NodeInterface;
 use Drupal\Core\Entity\Query\QueryFactory;
 use Drupal\Core\Entity\EntityTypeManagerInterface;
+
 /**
  * ContentApiClass
  */
 
 class ContentApiClass
 {
-    /**
-     * Node IDs
-     *
-     * @var array
-     */
-    protected $nid = array();
-    
-    /**
-     * Nodes
-     *
-     * @var array
-     */
-    protected $node = array();
-    /**
-     * Language Tag
-     *
-     * @var string
-     */
-    protected $lang;
-    /**
-     * Node_storage object
-     *
-     * @var Drupal\Core\Entity\EntityManagerInterface
-     */
-    protected $node_storage;
-    /**
-     * Entitity Query object
-     *
-     * @var Drupal\Core\Entity\Query\QueryFactory
-     * 
-     * Instance of querfactory 
-     */
-    protected $entity_query;
-    /**
-     * Class Constructor 
-     *
-     * @param EntityTypeManagerInterface $entityTypeManager
-     * @param QueryFactory $entityQuery
-     */
-    public function __construct(
-        EntityTypeManagerInterface $entityTypeManager, 
-        QueryFactory $entityQuery
-    ) {
-        $this->node_storage = $entityTypeManager->getStorage('node');
-        $this->entity_query = $entityQuery;
-    }
-    /**
-     * API resource function
-     *
-     * @param [string] $lang
-     * @return array
-     */
-    public function ContentApiEndpoint($lang, $nid)
-    {   
-        $this->lang = $lang;
```

Comment:
We need to re-add this line.. `translateNode` uses it..
I'd prefer `translateNode` to take it as an arg but let's be consistent :)